_inc/client/plans/plan-body.jsx

```diff
@@ -313,7 +313,9 @@ const PlanBody = React.createClass( {
 						</div>
 
 						<p>
-							<Button href={ 'https://jetpack.com/redirect/?source=plans-main-bottom&site=' + this.props.siteRawUrl } className="is-primary">
+							<Button href={ 'jetpack_free' === this.props.plans
```

Comment:
Is it ever possible that `this.props.plans` will be blank?  I'm thinking for old Jetpacks that have never gone though the new connection flow which forces you to choose a plan.